### PR TITLE
Add `pixlet login` for logging in via UI

### PR DESCRIPTION
**Pull request by rohansingh**

### CODE REVIEW

File: cmd/push.go [hunk 1]
```diff
@@ -37,8 +38,8 @@ func init() {
 }
 
 var PushCmd = &cobra.Command{
-	Use:   "push [device ID] [webp image] [installationID]",
-	Short: "Pushes a webp image to a Tidbyt device",
+	Use:   "push [device ID] [webp image]",
+	Short: "Run a Pixlet script and push the output to a Tidbyt",
```
Comment (matslina) [inline cmd/push.go:42]: This doesn't seem accurate.

**Review by matslina — APPROVED** [review]

File: cmd/push.go (hunk 1 above)
Comment (rohansingh) [reply cmd/push.go]: Updated to make it clearer.

File: cmd/login.go [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+	}
+
+	privateConfig.Set("token", tok)
+
+	if err := privateConfig.WriteConfig(); err != nil {
+		fmt.Println("persisting token:", err)
+		os.Exit(1)
+	}
+
+	say, err := cowsay.Say(
```
Comment (betterengineering) [inline cmd/login.go:128]: lol

File: cmd/login.go [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+		state,
+		handler,
+		pkce,
+	).Token()
+	if err != nil {
+		fmt.Println("getting auth token:", err)
+		os.Exit(1)
+	}
+
+	privateConfig.Set("token", tok)
```
Comment (betterengineering) [inline cmd/login.go:121]: 😍

**Review by betterengineering — APPROVED** [review]

**Pull request merged**